src/.nuget/packages.builds

```diff
@@ -40,13 +40,15 @@
       Overwrite="true" />
   </Target>
 
-  <!-- [ARCADE REMOVE] Make sure version props file for native binaries (non-Arcade versioning) exists
-       This is through building a separate project because the packaging projects use Arcade versioning
-       and just need the build number for laying out certain native binaries. These projects do not want
-       the actual BuildTools target to run / set various properties, so they just include the generated
-       build version props file (which only sets BuildNumberMajor/BuildNumberMinor) as needed. -->
-  <Target Name="CreateVersionPropsForNativeBinaries"
-          BeforeTargets="Build">
-    <MSBuild Targets="Build" Projects="$(CoreclrDir)\src\createVersionFile.proj"/>
+  <!-- Update the project references with additional properties calculated during the execution phase.
+       _InitializeAssemblyVersion is provided by Arcade. It sets the AssemblyVersion and FileVersion properties -->
+  <Target Name="UpdateAdditionalProperties"
+          BeforeTargets="Build"
+          DependsOnTargets="_InitializeAssemblyVersion">
```

Comment:
Just asking because I don't really enjoy taking dependency in private targets; in particular we took a dependency on this target's previous logic in diagnostics and at some point this changed. Is it possible to depend on `GetAssemblyVersion` which is the public Arcade target that's [documented](https://github.com/dotnet/arcade/blob/master/Documentation/CorePackages/Versioning.md#output)?

Reply:
I thought `GetAssemblyVersion` was one of the well-known targets in Microsoft.NET.Sdk (which packaging builds don't use), not Arcade. Maybe I'm missing something, but from what I can see it is defined by Microsoft.NET.GenerateAssemblyInfo.targets which gets pulled in with Microsoft.NET.Sdk, and Arcade just chains into it through BeforeTargets?

Reply:
Indeed it's part of the SDK. We massage it a little in arcade to adjust to what we need in the product, but in general their guidance was to hook into public targets. I have no strong feeling at all, just felt like it was worth asking if there was a reason not to to be aware of.

Reply:
Ah, okay. I'll add a comment to at least indicate that we are intentionally hooking into a private in this case.